### PR TITLE
Mangaworld

### PR DESCRIPTION
**Pull request by daarkdemon**
add two websites: [mangaworld](https://www.mangaworld.so/) and [mangaworldadult](https://www.mangaworldadult.com)

Checklist:
- [x] Updated source's version for individual source changes
- [x] Updated all sources' versions for template changes
- [x] Set appropriate `nsfw` value
- [x] Did not change `id` even if a source's name or language were changed
- [x] Tested the modifications by running it on the simulator or a test device

### CODE REVIEW

**Comment by daarkdemon** [comment]
Any update on this ????😅

**Comment by Skittyblock** [comment]
why did you update the readme? I don't think it should be changed like this (I'm not a fan of the badges), but even if it was to be changed, it should be done through a separate pr

File: src/rust/mangaworld/template/src/template.rs [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+		MangaContentRating::Safe
+	};
+	let manga_type = html
+		.select("div.info > div.meta-data.row.px-1 > div:nth-child(5) > a")
+		.text()
+		.read();
+	let viewer = match manga_type.as_str() {
+		"Manga" => MangaViewer::Rtl,
+		"Manhwa" => MangaViewer::Scroll,
+		_ => MangaViewer::Ltr,
```
Comment (Skittyblock) [inline src/rust/mangaworld/template/src/template.rs:155]: why are you falling back to ltr? you should use rtl for manga sources

**Review by Skittyblock — CHANGES_REQUESTED** [review]
and revert readme changes

File: src/rust/mangaworld/template/src/template.rs (hunk 1 above)
Comment (daarkdemon) [reply src/rust/mangaworld/template/src/template.rs:155]: It has mostly Chinese comics so it is left to right

**Comment by Skittyblock** [comment]
you still have readme changes

File: src/rust/mangaworld/template/src/template.rs (hunk 1 above)
Comment (Skittyblock) [reply src/rust/mangaworld/template/src/template.rs]: Manhua should be marked as scroll. I looked at the site, and the leftover types are like Doujinshi and Oneshot, which should also be Rtl.

**Review by Skittyblock — APPROVED** [review]

**Comment by bordam** [comment]
@daarkdemon Thank you so much!!! 😊 Can you share a link to your PayPal? Or can I donate to you in other ways?

**Pull request merged**